### PR TITLE
chore(metric): fix table id filter and some panel reorgs

### PR DESCRIPTION
**Pull request by hzxa21**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

- Fix table id filter after #11822 when empty string is used as the table id value.
- Remove some unused metrics: "Read Duration - MayExist", "preload ops".
- Avoid showing "streaming_upload_write_bytes|streaming_read_read_bytes|streaming_read" in object store rate and duration
- Separate the original Hummock section into two: Hummock (Read) and Hummock (Write)
- Avoid showing too detail percentile for historgram metrics. In most cases, p50, p99 and max should be sufficient.

## Checklist

- [ ] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [ ] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

File: grafana/common.py [hunk 1]
```diff
@@ -525,7 +526,8 @@ def metric(name, filter=None, node_filter_enabled=True, table_id_filter_enabled=
     if risingwave_name_filter_enabled:
         filters.append("risingwave_name=~\"$instance\"")
     if table_id_filter_enabled:
-        filters.append("table_id=~\"$table\"")
+        # We use "%table|" insated of "%table" here to match empty string for the table_id filter
```
Comment (zwang28) [inline grafana/common.py:529]: typo `insated`

**Review by zwang28 — APPROVED** [review]
LGTM.
unrelated to this PR, I had considered to replace some `row` with `row_collapsed` in grafana, to have a concise home page.

**Review by Li0k — APPROVED** [review]
LGTM, 

I'd like to split `Compaction Success & Failure Count` into two parts so that we can more easily observe changes in the number of failures

**Comment by hzxa21** [comment]
> LGTM. unrelated to this PR, I had considered to replace some `row` with `row_collapsed` in grafana, to have a concise home page.

Good point. I have made all the sections "row_collapsed".

**Comment by hzxa21** [comment]
> LGTM,
> 
> I'd like to split `Compaction Success & Failure Count` into two parts so that we can more easily observe changes in the number of failures

Good point. Done in the latest commit.

**Pull request merged**